sharpy/generators/dynamiccontrolsurface.py

```diff
@@ -2,66 +2,85 @@
 
 import sharpy.utils.generator_interface as generator_interface
 import sharpy.utils.settings as settings
+import sharpy.utils.cout_utils as cout_utils
 
 
 @generator_interface.generator
 class DynamicControlSurface(generator_interface.BaseGenerator):
     """
     Dynamic Control Surface deflection Generator
 
-    ``DynamicControlSurface`` class inherited from ``BaseGenerator``
+    The object generates a deflection in radians based on the time series given as a single vector in the input data.
+    A first order finite-differences scheme is used to calculate the deflection rate based on the provided time step
+    increment.
 
-    The object generates a deflection in radians based on the time series given as a vector in the input data
+    To call this generator, the ``generator_id = DynamicControlSurface`` key shall be used for the setting
+    `control_surface_deflection` in the ``AerogridLoader`` solver.
 
-    To call this generator, the ``generator_id = DynamicControlSurface`` shall be used.
-    This is parsed as the value for the ``control_surface_deflection_generator`` key in the aerogridloader solver's settings.
+    These generator settings should be parsed as a dictionary to the setting
+    ``control_surface_deflection_generator_settings`` in ``AerogridLoader``, where this dictionary should be the value
+    to another dictionary where the key is the index of the control surface in string format. This is shown better
+    in the example below:
```

Comment:
```suggestion
   One instance of this generator will be created for each control surface, thus, a group of settings should be defined 
   for each control surface (``cs0_settings``, ``cs1_settings`` ... in the example below).
   All of these groups of settings should be collected as values in a dictionary which keys are the associated control surface number in string format. 
   This dictionary should be parsed to the variable
    ``control_surface_deflection_generator_settings`` in ``AerogridLoader``. This is shown better
    in the example below:
```

I think it is more clear like this, but feel free to discard this comment